Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2015-2017 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM alpine:3.6
+MAINTAINER Stephen Schlie <schlie@tigera.io>
+
+ADD bin/confd /bin/confd
+
+RUN apk --no-cache add git ca-certificates openssl
+RUN update-ca-certificates
```

Comment:
I'm not 100% sure, I'd need to test it out but most folks seemed to believe that it did not do this during the install.  I only needed this for the `wget` commands which I went ahead and replaced with `ADD` as you recommended below.